notebooks/concepts/Specviz from splot concept notebook.ipynb

```diff
@@ -0,0 +1,3636 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "spec = specviz.get_viewer_data()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Which would be the same as the `current_spectrum` option above.\n",
+    "\n",
+    "In the below, `specviz.current_spectrum` is used for simplicity, but the intent is that that would instead be `get_viewer_data` if the above suggestion is followed."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## SPLOT commands addressed by data loading operations "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Options: images\n",
+    "\n",
+    "*Description*:\n",
+    "\n",
+    "The actual spectrum file that gets loaded when splot starts"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### g -- get another spectrum\n",
+    "\n",
+    "*Description*:\n",
+    "\n",
+    "The current spectrum is replaced by the\n",
+    "        new  spectrum.   The  aperture/line  and  band  are  queried  is \n",
+    "        necessary.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Query: next_image\n",
+    "\n",
+    "*Description*:\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Zoom/Pan\n",
+    "\n",
+    "The UI should support standard plot interaction options, including:\n",
+    "\n",
+    "* Zooming *vertically* (without zooming horizontally)\n",
+    "* Zooming *horizontally* (without zooming vertically)\n",
+    "* A \"reset to initial view\" option - i.e., show the whole spectrum.\n",
+    "* Box-select zooming (implicitly also panning)\n",
+    "* Standard \"hand\"/touch-style panning\n",
+    "* Center view on click/keystroke (optional/lower priority)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Explicily specifying limits\n",
+    "\n",
+    "There should also be a way to manually set the plot limits:\n",
+    "\n",
+    "1. In the notebook - e.g.:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "specviz.y_limits = (0, 1000) # <- uses whatever the current units are\n",
+    "\n",
+    "# note this is already available as part of the glue viewer, so this should be syntactic sugar for something like:\n",
+    "specviz.spectrum_viewer.state.y_min = 0\n",
+    "specviz.spectrum_viewer.state.y_max = 1000"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "    The notebook interface should also understand units.  That is, this should set the view to 5000 - 7000 angstroms regardless of what units the spectrum itself is in:"
+   ]
+  },
```

Comment:
Both the x_limits and y_limits should have default units matching spec, unless they are explicitly specified by tacking on units like '*u.Jy' or '*u.angstrom'

Reply:
I don't quite understand - what I was thinking here is that if you don't give units explicitly, it's implied to be "whatever units the spectrum is in", and the idea is that by loading via `Spectrum1D`, it *has* to have spectrum-compatible x and y units - so then it's not on `jdaviz` to worry about that but rather `specutils`.

Or did you mean you think we should have default units that are *not* the same as whatever that particular unit is if the user doesn't explicitly give units?

Reply:
You suggested always plotting in angstroms "regardless of what units the spectrum is in".  I disagree with that, but agree with your reply that specViz should plot in whatever units are attached to the Spec1D object.